### DISCUSSION TITLE
TVnfo "Television Information"

### DISCUSSION THREAD

**Discussion by ghost** [phalcon/phalcon#75] [Show and tell]
I've built a TVnfo project tvguide and schedule website on Phalcon.
A better version is coming sometime in 2021. https://tvnfo.com

**Comment by niden** [comment]
Thank you for this information. If you want to showcase your site, feel free to open a pull request in the [builtwith](https://github.com/phalcon/builtwith) repository

**Reply by ghost** [reply]
@niden created a pull request. https://github.com/phalcon/builtwith/pull/176
Also this issue related to link https://github.com/phalcon/builtwith/issues/177

**Comment by niden** [comment]
Resolved
